test/utils/trust.spec.ts

```diff
@@ -562,6 +563,44 @@ describe('Trust Utils method tests', () => {
     });
   });
 
+  describe('test if overseas entity requires manage trusts', () => {
+
+    test("test checkEntityRequiresManageTrusts with application data with no trusts to review", () => {
+
+      const result = checkEntityRequiresManageTrusts(mockAppData);
+      expect(result).toEqual(false);
+    });
+
+    test("test checkEntityRequiresManageTrusts with application data with trusts to review", () => {
+      const appData: ApplicationData = {
+        update: {
+          review_trusts: [
+            {
+              trust_id: "1",
+              trust_name: "test",
+              creation_date_day: "1",
+              creation_date_month: "2",
+              creation_date_year: "2023",
+              ch_reference: "test",
+              unable_to_obtain_all_trust_info: "false"
+            }
+          ]
+        }
+      };
+      const result = checkEntityRequiresManageTrusts(appData);
+      expect(result).toEqual(true);
+    });
+    test("test checkEntityRequiresManageTrusts with application data with no trusts to review", () => {
+      const appData: ApplicationData = {
+        update: {
+          review_trusts: []
+        }
+      };
+      const result = checkEntityRequiresManageTrusts(appData);
+      expect(result).toEqual(false);
+    });
```

Comment:
small but could you add a unit test to cover the review_trusts key doesn't exist in update as well just to make sure that's covered

Reply:
actually missed your first test there covers that

Reply:
what about a test for appdata = undefined? Maybe that's what Sonar is saying is uncovered

Reply:
fixed